### PR TITLE
Fill in the repository template gaps

### PR DESCRIPTION
**Pull request by celinval**
This PR fills in our initial group details. @pnkfelix

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,56 +9,39 @@
...
+
+This is the repository we use to organise our work. Please refer to our [charter] as well
 as our [github pages website][gh-pages] for more information on our goals and
 current scope.
 
 [charter]: ./CHARTER.md
-[gh-pages]: https://rust-lang.github.io/{{GROUP_SLUG}}
-
-**/FIX ME**
+[gh-pages]: https://rust-lang.github.io/project-stable-mir
```
Comment (pnkfelix) [inline README.md:28]: Do you happen to know what we need to do to get this link to start working? (Right now it 404's)

**Comment by pnkfelix** [comment]
This are good fixes. We can add more details later as we resolve our open questions.

File: README.md [hunk 2]
```diff
@@ -28,56 +9,39 @@
...
 
 
 ## How Can I Get Involved?
 
-**FIX ME**
-
-<!--
- List ways that people from outside your group can get involved and potentially
- become members, include what meetings your team has, and how a person could
```
[the remark this answers is not in the document]
Comment (celinval) [reply README.md:55]: That's fair. Thanks for kicking it off.

File: README.md (hunk 1 above)
Comment (celinval) [reply README.md:28]: I think we need to configure the github action. Let me fix that!

Comment (celinval) [reply README.md:28]: I published #3 to fix the workflow and I have also configured our Page section inside project Settings to use the root of gh-pages.

**Pull request merged**